### DISCUSSION TITLE
单数页、双数页的页码位置不一样？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#174] [Q&A]
大家有没有发现：单数页、双数页的页码位置不一样，页码数字不是居中的。怎么让页码剧中呢？

**Comment by sikouhjw** [comment]
你连截图都不给，哪来的说服力……

**Reply by earthmoon** [reply]
我仔细看了，页码是居中的，但是有些页码的位置确实不居中，这很奇怪。
![image](https://user-images.githubusercontent.com/45897471/191016530-847b06b9-3869-4509-af20-29d1fd3942ae.png)
![image](https://user-images.githubusercontent.com/45897471/191016876-43f4879b-9533-4e8a-b9ec-f796fd7beece.png)
这种差异出现有图片的页码，但是有图片的页码也不是全部这样。这个特征还不好把握。

**Reply by sikouhjw** [reply]
你试试能不能给出 MWE 吧……

**Reply by earthmoon** [reply]
> 你试试能不能给出 MWE 吧……

过些天，最近太忙了。

**Reply by earthmoon** [reply]
都忘了这个问题了，不好意思。我后面仔细排查了，是我修改了页面边距等，虽然我用了`restore`相关的命令恢复后面的页面设计，但是仍旧存在问题。后来我删除所有修改页边距的命令，替换为`adjustwidth`，这一问题就不存在了。谢谢！
